pod/perldelta.pod

```diff
@@ -268,6 +268,11 @@ have been sped up.
 
 C<grok_number_flags> has been sped up.
 
+=item *
+
+C<sort> is now noticeably faster in cases such as C<< sort {$a <=> $b} >> or
+C<< sort {$b <=> $a} >>.
```

Comment:
Is there any relevant issue link for this change?

Reply:
I'll add it.